scripts/localize_url.py

```diff
@@ -11,19 +11,33 @@ def get_files(path):
   return files
 
 def pprint(bad_links):
+  found = False
   for k,v in bad_links.items():
     if len(v) > 0:
       print(k)
       for vv in v:
         print("--> " + vv)
+        found = True
       print()
+  if (found == False):  
+    print("ok, no need to replace anything")
```

Comment:
you can actually simply add this simple check in line 14 and return early, len(bad_links) will give the total keys existed in a dict
i.e.
```
if (len(bad_links)) == 0:
  print("ok, no need to replace anything")
  return
```